### PR TITLE
Update mcr.cpp

### PR DESCRIPTION
**Pull request by lowen2022**
add one line note: new code here

### CODE REVIEW

**Comment by lowen2022** [comment]
hihi everyone~

File: mcr.cpp [hunk 1]
```diff
@@ -56,3 +56,8 @@ cout << game[1][0] << " " << game[1][1] << " " << game[1][2] << endl;
 cout << game[2][0] << " " << game[2][1] << " " << game[2][2] << endl;
 return 0;
 }
+//New code here~
```
Comment (lowen2022) [inline mcr.cpp:59]: I see

**Comment by tim-lyu** [comment]
Hello, Lowen. This is Tim. Nice to see you there.

File: mcr.cpp (hunk 1 above)
Comment (tim-lyu) [inline mcr.cpp:59]: Good job!

**Pull request merged**